src/v0/destinations/af/README.md

```diff
@@ -0,0 +1,332 @@
+
+- **Status Callback URLs**: Comma-separated URLs for GDPR request status callbacks (max 3 URLs)
+
+## Integration Functionalities
+
+> AppsFlyer supports both **Cloud mode** and **Device mode** connections
+
+### Supported Message Types
+
+#### Cloud Mode
+- Track
+- Page
+- Screen
+
+#### Device Mode
+- Track
+- Screen
+- Identify
+
+**Supported Platforms for Device Mode**: Android, iOS, React Native, Flutter, Cordova
+
+**Note**: Device mode uses the AppsFlyer native SDK and is only available for mobile platforms. Web, Unity, AMP, and server-side sources only support cloud mode.
+
+### Batching Support
+
+- **Supported**: Yes (cloud mode only)
+- **Message Types**: Track, Page, Screen events in cloud mode
+- **Implementation**: Uses `simpleProcessRouterDest` for concurrent processing
+- **Note**: This provides individual event processing with concurrency, not actual request batching. Device mode events are processed individually by the native SDK.
+
+### Rate Limits
+
+The AppsFlyer Server-to-Server Events API enforces the following rate limits:
+
+| API Version | Endpoint | Rate Limit | Payload Limit | Description |
+|-------------|----------|------------|---------------|-------------|
+| v2 (Legacy) | `https://api2.appsflyer.com/inappevent/{app_id}` | Not specified | Up to 1KB JSON payload | Legacy endpoint using Dev Key authentication |
+| v3 (Current) | `https://api3.appsflyer.com/inappevent/{app_id}` | Not specified | Up to 1KB JSON payload | Current endpoint using S2S Key authentication |
+
+**General Rate Limits:**
+- **OneLink API**: 500 requests per second (30,000 per minute)
+- **OpenDSR API**: 350 requests per minute (for user deletion)
+
+**Important Notes:**
+- JSON payload size is limited to 1KB per event
+- AppsFlyer supports gradual scaling for high-volume implementations
+- Recommended to start with 10K TPS and scale gradually with 1-minute intervals
+- Implement retry mechanisms for error handling
+
+[Docs Reference](https://dev.appsflyer.com/hc/reference/s2s-events-api3-post)
+
+### Intermediate Calls
+
+- **Supported**: No
+- **Description**: AppsFlyer destination does not make any intermediate API calls. All events are sent directly to the AppsFlyer Server-to-Server Events API.
+
+### Proxy Delivery
+
+- **Supported**: No
+- **Reason**: No `networkHandler.js` file is implemented for this destination
+
+### User Deletion
+
+- **Supported**: Yes
+- **Source Code Path**: `src/v0/destinations/af/deleteUsers.js`
+- **API Used**: AppsFlyer OpenDSR API for GDPR compliance
+- **Endpoint**: `https://hq1.appsflyer.com/api/gdpr/v1/opendsr_requests`
+- **Authentication**: Requires API Token (Bearer token)
+- **Supported Identity Types**:
+  - `appsflyer_id` (highest priority)
+  - `ios_advertising_id` (requires Apple App ID)
+  - `android_advertising_id` (requires Android App ID)
+- **Request Type**: Erasure requests following OpenDSR protocol
+- **Rate Limit**: 350 requests per minute
```

Comment:
Can we also put this somehow, it is an important limitation: 
Checking the status of a request from more than 60 days ago returns "request not found".